apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -581,12 +566,10 @@ contract DisputableVoting is DisputableApp {
             return false;
         }
 
-        // If the vote cannot proceed due to an Agreement dispute, it cannot be executed
-        if (!_canProceedAgreementAction(vote_.actionId)) {
-            return false;
-        }
-
-        // If none of the above conditions are met, it can be executed
+        // If none of the above conditions are met, it can be executed.
+        // We are not checking if the vote can proceed against the Agreement app since it will
+        // be tried to be closed when executed. This means the close call will fail in the Agreement
+        // side if the vote's status cache on the voting app was outdated for some weird reason.
```

Comment:
```suggestion
        // At this point, we are not checking if the vote can proceed against the Agreement app.
        // This will be done on the vote's execution, as it tries to close the action with the Agreement app.
        // This means the close call can fail in the Agreement's side if the status cache here was outdated
        // for some weird reason.
```